homeassistant/components/weheat/manifest.json

```diff
@@ -6,5 +6,5 @@
   "dependencies": ["application_credentials"],
   "documentation": "https://www.home-assistant.io/integrations/weheat",
   "iot_class": "cloud_polling",
-  "requirements": ["weheat==2025.2.26"]
+  "requirements": ["weheat==2025.2.27"]
```

Comment:
Please bump this in a separate PR

Reply:
Done in https://github.com/home-assistant/core/pull/142539